nni/retiarii/oneshot/pytorch/proxyless.py

```diff
@@ -136,7 +208,10 @@ class ProxylessTrainer(BaseOneShotTrainer):
     def __init__(self, model, loss, metrics, optimizer,
                  num_epochs, dataset, warmup_epochs=0,
                  batch_size=64, workers=4, device=None, log_frequency=None,
-                 arc_learning_rate=1.0E-3):
+                 arc_learning_rate=1.0E-3,
+                 grad_reg_loss_type=None, grad_reg_loss_params=None,
```

Comment:
Please add docstrings for newly-added parameters.

Reply:
Have fixed other comments. The doc will commit tonight. Thanks a lot!